directed_forgetting/experiment.js

```diff
@@ -66,6 +66,48 @@ var getInstructFeedback = function() {
 		'</p></div>'
 }
 
+var getTestFeedback = function() {
+	var global_trial = jsPsych.progress().current_trial_global
+	var data_length = 144
+	var start_cut = global_trial - data_length
+	
+	//below are counters to see if the subject is treating this task as a directed remembering as opposed to a directed forgetting task
+	var neg_respond_remember_total = 0
+	var pos_respond_remember_total = 0
+	var neg_respond_remember = 0
+	var pos_respond_remember = 0
+	
+	for(var b =0; b < data_length; b++){
+		if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).probe_type == 'neg'){
+		neg_respond_remember_total += 1
+			if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).key_press == '37'){
+			neg_respond_remember += 1
+			}
+		}else if (jsPsych.data.getDataByTrialIndex(start_cut + b - 1).probe_type == 'pos'){
+		pos_respond_remember_total += 1
+			if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).key_press == '39'){
+			pos_respond_remember_total += 1
+			}
+		}
+	}
+	
+	var directed_remembering_total = neg_respond_remember + pos_respond_remember
+	var directed_remembering_percent = directed_remembering_total / pos_respond_remember_total + neg_respond_remember_total
```

Comment:
I don't think you are calculating this right. Just calculate it as (pos_respond_remember+neg_respond_remember)/total_responses